<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepNeuronDialog</class>
 <widget class="QDialog" name="DeepNeuronDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>331</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deep Neuron</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>680</width>
        <height>307</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Caffe</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="title">
              <string>Select required files</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Training dataset mean (.binaryproto):</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="pushButton_3">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QPushButton" name="pushButton_5">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="4">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Model deployment file (.prototxt): </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Trained model file (.caffemodel):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="pushButton_4">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="checkBox">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Save as default settings</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start Labeling</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DeepNeuronDialog</receiver>
   <slot>okClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okClicked()</slot>
 </slots>
</ui>
